docs/book/src/topics/disk-encryption.md

```diff
@@ -0,0 +1,55 @@
+# Disk Encryption
+This document describes how to configure different encryption options for disks allocated to VMs provisioned in Azure. 
+
+## Azure Disk Storage Server-Side Encryption
+Azure Disk Storage Server-Side Encryption (SSE) is also referred to as encryption-at-rest. This encryption option does not encrypt temporary disks or disk caches.
+
+When enabled, Azure Disk Storage SSE encrypts data stored on Azure managed disks, i.e. OS and data disks. This option can be enabled using customer managed keys.
+
+Customer managed keys must be configured through a Disk Encryption Set (DES) resource. For more information on Azure Disk Storage SSE, please see this [link](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption).
```

Comment:
```suggestion
Customer-managed keys must be configured through a Disk Encryption Set (DES) resource. For more information on Azure Disk Storage SSE, please see this [link](https://learn.microsoft.com/azure/virtual-machines/disk-encryption).
```
This is nitpicky, but I try to remove the `en-us/` part of doc URLs so the user can access a localized version depending on their language preferences.

Reply:
That makes total sense. I didn't even think about that. I'll try to remember that for next time :)